dataprofiler/profilers/profiler_options.py

```diff
@@ -352,6 +352,8 @@ def __init__(self) -> None:
         :vartype num_zeros: BooleanOption
         :ivar num_negatives: boolean option to enable/disable num_negatives
         :vartype num_negatives: BooleanOption
+        :ivar num_quantiles: boolean option to enable/disable num_quantiles
```

Comment:
need to add tests for the addition to options

Reply:
or update tests

Reply:
yes, need update to options testing for this